examples/iam-access-groups.v2.test.js

```diff
@@ -168,7 +146,15 @@ describe('IamAccessGroupsV2', () => {
     // begin-list_access_groups
 
     const params = {
-      accountId: testAccountId,
+      accountId: 'testString',
```

Comment:
Working examples code should not be using the mock values generated by the SDK generator (e.g. "testString").  Either manually change the examples after generation or add example values for various properties/parameters to your API definition so that those example values are used during generation of integration tests and examples.